spanner/test/cloudexecutor/executor/actions/batch.go

```diff
@@ -50,10 +52,16 @@ func (h *StartBatchTxnHandler) ExecuteAction(ctx context.Context) error {
 		return h.OutcomeSender.FinishWithError(spanner.ToSpannerError(status.Error(codes.InvalidArgument, "database path must be set for this action")))
 	}
 
-	client, err := spanner.NewClient(ctx, h.FlowContext.Database, h.Options...)
+	client, err := spanner.NewClientWithConfig(ctx, h.FlowContext.Database, spanner.ClientConfig{SessionPoolConfig: spanner.DefaultSessionPoolConfig, DisableRouteToLeader: false, EnableEndToEndTracing: true}, h.Options...)
+	if err != nil {
+		return h.OutcomeSender.FinishWithError(err)
+	}
+	// Create a trace client to read the traces from Cloud Trace.
+	traceClient, err := trace.NewClient(ctx, h.TraceClientOptions...)
```

Comment:
Looks like the traceClient is independent of the type of action and can be initialized just once at the beginning of the `Execute()` method - [here](https://github.com/googleapis/google-cloud-go/blob/a187451a912835703078e5b6a339c514edebe5de/spanner/test/cloudexecutor/executor/internal/inputstream/handler.go#L62).
Remove it from `StartBatchTxnHandler` and `StartTxnHandler`.

Reply:
Done.